pkgs/build-support/fetchurl/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, curl }: # Note that `curl' may be `null', in case of the native stdenv.
+{ stdenv, curl, ca_cert_file ? null, insecure ? false }: # Note that `curl' may be `null', in case of the native stdenv.
```

Comment:
Anyway does this look good otherwise?

On Sun, May 31, 2015, 01:32 Vladimír Čunát <notifications@github.com> wrote:

> In pkgs/build-support/fetchurl/default.nix
> <https://github.com/NixOS/nixpkgs/pull/8082#discussion_r31388163>:
>
> > @@ -1,4 +1,4 @@
> > -{ stdenv, curl }: # Note that `curl' may be `null', in case of the native stdenv.
> > +{ stdenv, curl, ca_cert_file ? null, insecure ? false }: # Note that `curl' may be `null', in case of the native stdenv.
>
> I think this is the relevant place: NixOS/nix#491
> <https://github.com/NixOS/nix/pull/491>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/8082/files#r31388163>.
>